src/main/kotlin/no/ssb/metadata/vardef/integrations/vardok/services/VardokApiService.kt

```diff
@@ -18,7 +18,8 @@ open class VardokApiService(
 ) : VardokService {
     private val logger = LoggerFactory.getLogger(VardokApiService::class.java)
 
-    override fun isDuplicate(name: String): Boolean = variableDefinitionRepository.existsByShortName(name)
+    override fun isDuplicate(name: String): Boolean =
+        variableDefinitionRepository.existsByShortName(name.lowercase().replace("""[-\s]""".toRegex(), "_"))
```

Comment:
This is some surprising looking magic, it would be good to have a comment or even to encapsulate it into a function so it's clear what we're trying to achieve